pimcore/modules/admin/views/scripts/index/index6.php

```diff
@@ -575,7 +576,6 @@
         showCloseConfirmation: true,
         debug_admin_translations: <?= \Zend_Json::encode((bool) $this->config->general->debug_admin_translations) ?>,
         document_generatepreviews: <?= \Zend_Json::encode((bool) $this->config->documents->generatepreview) ?>,
-        asset_disable_tree_preview: <?= \Zend_Json::encode((bool) $this->config->assets->disable_tree_preview) ?>,
```

Comment:
This needs to be fixed. This was added by  2db3254afbfd393bb3b1ff44962e55a5fc2b14fa 
Thanks!